src/styles/card_styles/card.scss

```diff
@@ -0,0 +1,128 @@
+.cards {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), avoid using fixed widths for grid-template-columns. Instead, use max-width or percentages to make it more flexible and maintainable.